templates/first-snap/build.html

```diff
@@ -27,6 +27,14 @@ <h4 class="p-list-step__title">
               {% endif %}
             </li>
           {% endfor %}
+          {% if os.startswith('linux') %}
```

Comment:
It feels a bit weird to be hardcoding this warnings in the template while they depend on the content (existence of 'offlineimap' in commands).  If the content changes we may end up with showing unnecessary warnings (or not showing warnings when needed). We should find another way to do it.

Maybe there should be some field in content .yaml to turn the warning on? Or the content could be written in a way that avoids the need for them?